gads-ui/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "gads-ui",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://0.0.0.0:10000/",
```

Comment:
Don't commit a preset proxy value, in general people shouldn't even need it unless they develop on the UI